cgmanifest.json

```diff
@@ -1678,6 +1678,32 @@
       },
       "license": "The Apache Software License, Version 2.0"
     },
+    {
+      "component": {
+        "type": "maven",
+        "maven": {
+          "artifactId": "health-services-client",
+          "groupId": "androidx.health",
+          "version": "1.0.0-rc02",
+          "nuGetId": "Xamarin.AndroidX.Health.ServicesClient",
+          "nuGetVersion": "1.0.0-rc02"
+        }
+      },
+      "license": "The Apache Software License, Version 2.0"
+    },
+    {
+      "component": {
+        "type": "maven",
+        "maven": {
+          "artifactId": "connect-client",
+          "groupId": "androidx.health.connect",
+          "version": "1.0.0-alpha11",
```

Comment:
why not use the lates version ? [1.1.0-alpha07](https://mvnrepository.com/artifact/androidx.health.connect/connect-client/1.1.0-alpha07)